CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/DBHelperClass.java

```diff
@@ -8,6 +8,9 @@
 
 import androidx.annotation.Nullable;
 
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
```

Comment:
Might have added these by mistake?

Reply:
Ooh its great you spotted those. Will remove the right away.